scripts/mtls/disable_mtls.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+set -e
+
+help() {
+  echo ""
+  echo "Usage: $0 [environment]"
```

Comment:
Just pop a message in here to help explain what this tool is for and what it does. I might not remember in a few months time. It would help to include some context about why we have this.